core/network/src/main/java/com/school_of_company/network/util/TokenAuthenticator.kt

```
@@ -1,15 +1,67 @@
+            // 새롭게 발급을 받은 토큰을 DataSource에 저장하고, 이후 요청에 사용하도록 합니다.
+            runBlocking {
+                with(authTokenDataSource) {
+                    setAccessToken(respone.accessToken)
+                    setAccessTokenExp(respone.accessTokenExpiresIn)
+                    setRefreshToken(respone.refreshToken)
+                    setRefreshTokenExp(respone.refreshTokenExpiresIn)
+                }
+            }
+
+            respone.accessToken
+        } catch (e: Exception) {
```

detekt

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)